### PR TITLE
feat(`crosschain`): `MsgAbortStuckCCTX` message

### PR DESCRIPTION
**Pull request by lumtis**
# Description

Implement `MsgAbortStuckCCTX` that allows admin group 2 to abort a cctx in pending status

### CODE REVIEW

File: x/crosschain/client/cli/cli_cctx.go [hunk 1]
```diff
@@ -277,3 +277,25 @@ func CmdCCTXOutboundVoter() *cobra.Command {
...
+		Short: "abort a stuck CCTX",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+
+			msg := &types.MsgAbortStuckCCTX{
+				Creator:   clientCtx.GetFromAddress().String(),
+				CctxIndex: args[0],
+			}
+
```
Comment (kingpinXD) [inline x/crosschain/client/cli/cli_cctx.go:293]: maybe add validatebasic here

File: x/crosschain/keeper/msg_server_abort_stuck_cctx_test.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+		k, ctx, _, zk := keepertest.CrosschainKeeper(t)
+		msgServer := crosschainkeeper.NewMsgServerImpl(*k)
+		admin := sample.AccAddress()
+		setAdminPolicies(ctx, zk, admin)
+
+		// create a cctx
+		cctx := sample.CrossChainTx(t, "cctx_index")
+		cctx.CctxStatus = &crosschaintypes.Status{
+			Status:        crosschaintypes.CctxStatus_OutboundMined,
+			StatusMessage: "outbound minted",
```
Comment (kingpinXD) [inline x/crosschain/keeper/msg_server_abort_stuck_cctx_test.go:140]:
```suggestion
			StatusMessage: "outbound mined",
```

**Review by kingpinXD — APPROVED** [review]
Lgtm , minor comments

**Review by brewmaster012 — APPROVED** [review]

**Review by ws4charlie — APPROVED** [review]

File: x/crosschain/client/cli/cli_cctx.go (hunk 1 above)
Comment (lumtis) [reply x/crosschain/client/cli/cli_cctx.go:293]: It get called in `GenerateOrBroadcastTxCLI` actually

**Pull request merged**